### ISSUE TITLE
[ML on GCP C8] Image Classification with a Deep Neural Network Model -> only one entry in Tensorboard after training

### ISSUE THREAD

**Issue by tarrade** [GoogleCloudPlatform/training-data-analyst#403]
Great tutorial. One thing may have changed since them because in TensorBoard only one point is now display for accuracy and loss:

<img width="833" alt="Screenshot 2019-03-13 at 08 20 18" src="https://user-images.githubusercontent.com/12021701/54262622-ef98d300-456e-11e9-80e5-dc15bb254d13.png">

This is not what is display on the correction of this lab. Thanks

**Comment by tarrade** [comment]
With the new version of Tensorflow 2.0, Tensorbaord running in notebook, the new notebook instance instead of datalab, everything will need to be updated soon and my comment are then obsolete. Looking forward the updated version with all the changes mentioned above. Thanks.
